### PR TITLE
Adds Mime Goblins

### PR DESCRIPTION
**Pull request by Kharkov72**
Untitled
Behold, the slightly less annoying but still annoying counterpart of the clown goblin, the mime goblin.
Instead of slipping you, it silences you (which stacks) and makes mime walls.
It is made exactly how you think it would be made.
This also adds the cluwne book-gibbing to faguettes (oversight) and tweaks its emotes to not be weird for the mime goblin to make (like how clown goblins cry and such)
[content]

:cl:
 * rscadd: Adds Mime Goblins!

### CODE REVIEW

**Comment by N3X15** [comment]
It doesn't spam false walls?  **DENIED**

**Comment by Kharkov72** [comment]
the nightmare beings have spoken, they now make invis walls  at a 10% chance.
Untitled

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 1]
```diff
@@ -58,6 +58,10 @@
 		return
 	..()
 
+/mob/living/simple_animal/hostile/retaliate/faguette/attackby(obj/item/weapon/O, mob/user)
+	if(istype(O,/obj/item/weapon/book))
+		gib()
+		return
```
Comment (N3X15) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm]: This `return` might not be necessary.

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 2]
```diff
@@ -68,5 +72,58 @@
 	if(timestopped)
 		return //under effects of time magick
 
-	var/msg = pick("drools through its stitched mouth","silently cries into its baguette","sloppily mimes tieing an invisible noose around its neck")
+	var/msg = pick("drools slightly","mimes crying into a tissue","sloppily mimes tieing an invisible noose")
```
Comment (N3X15) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm]:
```suggestion
	var/msg = pick("drools slightly","mimes crying into a tissue","sloppily mimes tying an invisible noose")
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 3]
```diff
@@ -68,5 +72,58 @@
...
+		if(prob(10))
+			L.silent += 3
+			L.visible_message("<span class='danger'>\The [src.name] silences \the [L.name]!</span>")
+			return
+	return ..()
+
+
+/mob/living/simple_animal/hostile/retaliate/faguette/goblin
+	name = "mime goblin"
+	desc = "A tiny talking beret and gloves. Is it miming for a baguette?"
```
Comment (DrSnips) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm]: TALKING mimes?

**Review by N3X15 — CHANGES_REQUESTED** [review]

**Comment by Kharkov72** [comment]
also since the return isn't needed someone (maybe me but unatomic) can also remove it from cluwne.dm

**Comment by MadmanMartian** [comment]
I had something like this on another branch. It would only be visible to those it's hunting, and only in close proximity

So, you'd see somebody getting beaten by an unseen foe, truly the mime experience

**Comment by Shadowmech88** [comment]
With a sprite and animation like that, I feel like it's meant to be accompanied by a sound like this.
https://www.youtube.com/watch?v=-hmNVq6HUfQ

**Comment by Kharkov72** [comment]
They now slap around, thus it is decreed.

**Comment by Kharkov72** [comment]
>animal.dmi gets smaller
🤔 

I checked and made sure no sprites were getting yeeted into the void though.

**Comment by GermanSanta** [comment]
This brings up the age old debate. Should mimes ever make a sound?

**Comment by Zth--** [comment]
> I checked and made sure no sprites were getting yeeted into the void though.

good enough

**Pull request merged**